sys/include/usb/usbus/cdc/ecm.h

```diff
@@ -77,6 +77,11 @@ extern "C" {
  */
 #define USBUS_CDCECM_EP_DATA_SIZE  64
 
+/**
+ * @brief Full ethernet frame rounded up to a whole number of transfers
+ */
+#define USBUS_ETHERNET_FRAME_BUF    (1536)
```

Comment:
```suggestion
#define USBUS_ETHERNET_FRAME_BUF    MATH_ALIGN(ETHERNET_FRAME_LEN, USBUS_CDCECM_EP_DATA_SIZE)
```

#17702 would be handy here

Reply:
Rebased and fixed